app/templates/settings/organizations/index.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="auk-scroll-wrapper__body au-u-background-gray-100">
+  <div class="auk-o-grid au-o-grid--stretch">
+    <div class="auk-o-grid-col-2">
+    </div>
+    <div class="auk-o-grid-col-10">
+      <DataTable
+        @content={{@model}}
+        @class="auk-table auk-table--styled auk-table--bordered auk-table--noactions"
+        @page={{this.page}}
+        @size={{this.size}}
+        @sort={{this.sort}}
+        @noDataMessage={{t "no-results-found"}}
+        as |table|
+      >
+        <table.content as |c|>
+          <c.header>
+            <ThSortable
+              @currentSorting={{this.sort}}
+              @field="name"
+              @label={{t "organization"}}
+            />
+            <th>{{t "ovo-code"}}</th>
+            <th>{{t "number-of-users"}}</th>
+          </c.header>
+          <c.body as |organization|>
+            <td>
+              <span class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-small">
+                {{organization.name}}
+              </span>
+            </td>
+            <td>
+              <span class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-small">
+                {{organization.identifier}}
+              </span>
+            </td>
+            <td>
+              <div>
+                <span class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-small">
+                  {{organization.memberships.length}} {{t "users"}}
```

Comment:
Dit gaat geen correct cijfer zijn als er meer members zijn dan de max page size (20 by default denk ik) want relatie wordt gepagineerd opgehaald.

2 mogelijkheden om op te lossen: 
1. memberships mee includen in de request in de model-hook. Zou ik vermijden wegens veel te veel irrelevante data die mee geladen wordt 
2. aparte component die een `this.store.count` doet op memberships voor een gegeven organisatie. Data verschijnt dan gradueel/iets later, maar lijkt me geen probleem voor deze case.

Ik zou voor optie 2 gaan.

Reply:
Merci, had hier niet aan gedacht. Optie 2 lijkt me ook ideaal, vooral als we een kleine loader toevoegen (wat we imo op meer plekken zouden mogen doen in de app)